examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/page-authoring/link/.content.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
link component should be part of "Adaptive Form" group in the component library ? From your changes it looks part of "Page Authoring" (aka) Core Content group.

Reply:
Element is not visible at top level if namespaced with Adaptive Form